src/syzygy/syzygy.cc

```diff
@@ -1618,7 +1618,7 @@ int SyzygyTablebase::probe_dtz(const Position& pos, ProbeState* result) {
 // Use the DTZ tables to rank root moves.
 //
 // A return value false indicates that not all probes were successful.
```

Comment:
Outdated comment.
But actually, return value was intended to return error and now it just is let go?..

Reply:
This is strange. My file has the correct comment: 
```// Use the DTZ tables to rank root moves.
// A return value 0 indicates that not all probes were successful.
// Otherwise best rank is returned, with rank 1 = draw.```